unfetter-discover/unfetter-store
# Added latest and latest by creator APIs for object assessments

Fixes unfetter-discover/unfetter#885

Renamed some files for naming consistency, and added created-by-ref to object assessments also for consistency with other objects.

# To test:
1. Pull this PR
2. Run 'npm run test' inside unfetter-store/unfetter-discover-api to verify succesful tests
3. Restart stack
4. Navigate to API Explorer and test out the object assessments '/v3/x-unfetter-object-assessments/latest' and '/v3/x-unfetter-object-assessments/latest/{creatorId}' APIs
    a. You'll need to add a 'creator' attribute to one of the sample object assessments.  You can do this manually in Robo3T and use your own identity ID from the Users collection.
